starknet_py/net/schemas/rpc.py

```diff
@@ -105,12 +105,14 @@ class TransactionSchema(Schema):
     hash = Felt(data_key="transaction_hash", required=True)
     signature = fields.List(Felt(), data_key="signature", load_default=[])
     max_fee = Felt(data_key="max_fee", load_default=0)
+    version = Felt(data_key="version", load_default=0)
```

Comment:
I'm not sure about this. Are there cases where the versions is missing according to the specs? So we need this `load_default`.

Reply:
Yeah, you're right. Will fix that